scripts/tasks/check-and-rewrite-package-json.js

```diff
@@ -36,6 +36,9 @@ const LWC_EXPOSED_MODULES = {
         // TODO [#3517]: remove support for deprecated 'wire-service' import
         'wire-service',
     ],
+    // This is an experimental package and is not included by default as part of
+    // @lwc/rollup-plugin/src/index.tx
+    '@lwc/signals': ['@lwc/signals'],
```

Comment:
You do not need this here I believe. This `LWC_EXPOSED_MODULES` is just for the module resolution system used in `lwc.config.json`. I don't think we want to keep adding stuff to that, because we probably want to eventually just rely on the built-in Node module resolution system anyway.